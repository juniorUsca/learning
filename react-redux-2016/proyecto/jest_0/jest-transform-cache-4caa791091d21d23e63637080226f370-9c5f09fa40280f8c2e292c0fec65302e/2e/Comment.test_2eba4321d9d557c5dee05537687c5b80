1b4cb02d38201d97c68ebc48c501ccd8
'use strict';var _react = require('react');var _react2 = _interopRequireDefault(_react);
var _reactTestRenderer = require('react-test-renderer');var _reactTestRenderer2 = _interopRequireDefault(_reactTestRenderer);
var _reactIntl = require('react-intl');

var _Comment = require('./Comment');var _Comment2 = _interopRequireDefault(_Comment);
var _messages = require('../../messages.json');var _messages2 = _interopRequireDefault(_messages);function _interopRequireDefault(obj) {return obj && obj.__esModule ? obj : { default: obj };}

const comment = {
  id: 1,
  email: 'cursos@platzi.com',
  name: 'Platzi Team',
  body: 'Este es un comentario de prueba' };


test('Comment should render the component', () => {
  const tree = _reactTestRenderer2.default.create(
  _react2.default.createElement(_reactIntl.IntlProvider, { locale: 'es', messages: _messages2.default.es },
    _react2.default.createElement(_Comment2.default, comment))).

  toJSON();

  expect(tree).toMatchSnapshot();
});
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkNvbW1lbnQudGVzdC5qc3giXSwibmFtZXMiOlsiY29tbWVudCIsImlkIiwiZW1haWwiLCJuYW1lIiwiYm9keSIsInRlc3QiLCJ0cmVlIiwiY3JlYXRlIiwiZXMiLCJ0b0pTT04iLCJleHBlY3QiLCJ0b01hdGNoU25hcHNob3QiXSwibWFwcGluZ3MiOiJhQUFBLDhCO0FBQ0Esd0Q7QUFDQTs7QUFFQSxvQztBQUNBLCtDOztBQUVBLE1BQU1BLFVBQVU7QUFDZEMsTUFBSSxDQURVO0FBRWRDLFNBQU8sbUJBRk87QUFHZEMsUUFBTSxhQUhRO0FBSWRDLFFBQU0saUNBSlEsRUFBaEI7OztBQU9BQyxLQUFLLHFDQUFMLEVBQTRDLE1BQU07QUFDaEQsUUFBTUMsT0FBTyw0QkFBU0MsTUFBVDtBQUNYLDJEQUFjLFFBQU8sSUFBckIsRUFBMEIsVUFBVSxtQkFBU0MsRUFBN0M7QUFDRSxxREFBYVIsT0FBYixDQURGLENBRFc7O0FBSVhTLFFBSlcsRUFBYjs7QUFNQUMsU0FBT0osSUFBUCxFQUFhSyxlQUFiO0FBQ0QsQ0FSRCIsImZpbGUiOiJDb21tZW50LnRlc3QuanN4Iiwic291cmNlc0NvbnRlbnQiOlsiaW1wb3J0IFJlYWN0IGZyb20gJ3JlYWN0JztcbmltcG9ydCByZW5kZXJlciBmcm9tICdyZWFjdC10ZXN0LXJlbmRlcmVyJztcbmltcG9ydCB7IEludGxQcm92aWRlciB9IGZyb20gJ3JlYWN0LWludGwnO1xuXG5pbXBvcnQgQ29tbWVudCBmcm9tICcuL0NvbW1lbnQnO1xuaW1wb3J0IG1lc3NhZ2VzIGZyb20gJy4uLy4uL21lc3NhZ2VzLmpzb24nO1xuXG5jb25zdCBjb21tZW50ID0ge1xuICBpZDogMSxcbiAgZW1haWw6ICdjdXJzb3NAcGxhdHppLmNvbScsXG4gIG5hbWU6ICdQbGF0emkgVGVhbScsXG4gIGJvZHk6ICdFc3RlIGVzIHVuIGNvbWVudGFyaW8gZGUgcHJ1ZWJhJyxcbn07XG5cbnRlc3QoJ0NvbW1lbnQgc2hvdWxkIHJlbmRlciB0aGUgY29tcG9uZW50JywgKCkgPT4ge1xuICBjb25zdCB0cmVlID0gcmVuZGVyZXIuY3JlYXRlKFxuICAgIDxJbnRsUHJvdmlkZXIgbG9jYWxlPVwiZXNcIiBtZXNzYWdlcz17bWVzc2FnZXMuZXN9PlxuICAgICAgPENvbW1lbnQgey4uLmNvbW1lbnR9IC8+XG4gICAgPC9JbnRsUHJvdmlkZXI+LFxuICApLnRvSlNPTigpO1xuXG4gIGV4cGVjdCh0cmVlKS50b01hdGNoU25hcHNob3QoKTtcbn0pOyJdfQ==