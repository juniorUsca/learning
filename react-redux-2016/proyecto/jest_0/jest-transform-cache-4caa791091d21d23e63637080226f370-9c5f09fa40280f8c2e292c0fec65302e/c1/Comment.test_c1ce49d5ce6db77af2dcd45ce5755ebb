5e08c642ccc6f0436ede6ed4ba05da89
'use strict';var _react = require('react');var _react2 = _interopRequireDefault(_react);
var _reactTestRenderer = require('react-test-renderer');var _reactTestRenderer2 = _interopRequireDefault(_reactTestRenderer);

var _Comment = require('./Comment.jsx');var _Comment2 = _interopRequireDefault(_Comment);function _interopRequireDefault(obj) {return obj && obj.__esModule ? obj : { default: obj };}


test('Comment should render the component', () => {
  const tree = _reactTestRenderer2.default.create(_react2.default.createElement(_Comment2.default, null)).toJSON();

  expect(tree).toMatchSnapshot();
});
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkNvbW1lbnQudGVzdC5qc3giXSwibmFtZXMiOlsidGVzdCIsInRyZWUiLCJjcmVhdGUiLCJ0b0pTT04iLCJleHBlY3QiLCJ0b01hdGNoU25hcHNob3QiXSwibWFwcGluZ3MiOiJhQUFBLDhCO0FBQ0Esd0Q7O0FBRUEsd0M7OztBQUdBQSxLQUFLLHFDQUFMLEVBQTRDLE1BQU07QUFDaEQsUUFBTUMsT0FBTyw0QkFBU0MsTUFBVCxDQUFnQixzREFBaEIsRUFBNkJDLE1BQTdCLEVBQWI7O0FBRUFDLFNBQU9ILElBQVAsRUFBYUksZUFBYjtBQUNELENBSkQiLCJmaWxlIjoiQ29tbWVudC50ZXN0LmpzeCIsInNvdXJjZXNDb250ZW50IjpbImltcG9ydCBSZWFjdCBmcm9tICdyZWFjdCc7XG5pbXBvcnQgcmVuZGVyZXIgZnJvbSAncmVhY3QtdGVzdC1yZW5kZXJlcic7XG5cbmltcG9ydCBDb21tZW50IGZyb20gJy4vQ29tbWVudC5qc3gnO1xuXG5cbnRlc3QoJ0NvbW1lbnQgc2hvdWxkIHJlbmRlciB0aGUgY29tcG9uZW50JywgKCkgPT4ge1xuICBjb25zdCB0cmVlID0gcmVuZGVyZXIuY3JlYXRlKDxDb21tZW50IC8+KS50b0pTT04oKTtcblxuICBleHBlY3QodHJlZSkudG9NYXRjaFNuYXBzaG90KCk7XG59KTsiXX0=