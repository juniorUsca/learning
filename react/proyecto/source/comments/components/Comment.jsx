import React from 'react';

// es una funcion pura xq no se hace peticiones ni nada

function Comment (props) {
  return(
    <article id={`comment-${props.id}`}>
      <div>
        By: <a href={`mailto: ${props.email}`}>{props.name}</a>
      </div>
      <p>{props.body}</p>
    </article>
  );
}

export default Comment;
