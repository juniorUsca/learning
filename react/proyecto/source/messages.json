{
  "en": {
    "error.404": "Go back to home",
    "title": "My blog in react",
    "title.home": "Homeee",
    "title.profile": "Profile of {name}",
    "profile.field.basic": "Basic info",
    "profile.field.address": "Address",
    "profile.title": "Profile",
    "header.nav.home": "Home",
    "comment.meta.author": "By: <a href=\"mailto:{email}\" target=\"_blank\">{name}</a>",
    "post.meta.comments": "{amount} comments",
    "post.meta.readMore": "Read more...",
    "loading": "Loading data..."
  },
  "es": {
    "error.404": "Volver al inicio",
    "title": "Mi blog en react",
    "title.home": "Inicio",
    "title.profile": "Perfil de {name}",
    "profile.field.basic": "Informaci칩n b치sica",
    "profile.field.address": "Direcci칩n",
    "profile.title": "Perfil",
    "header.nav.home": "Inicio",
    "comment.meta.author": "Por: <a href=\"mailto:{email}\" target=\"_blank\">{name}</a>",
    "post.meta.comments": "{amount} comentarios",
    "post.meta.readMore": "Leer m치s...",
    "loading": "Cargando datos..."
  }
}
