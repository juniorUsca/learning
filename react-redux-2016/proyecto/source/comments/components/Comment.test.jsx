import React from 'react';
import renderer from 'react-test-renderer';
import { IntlProvider } from 'react-intl';

import Comment from './Comment';
import messages from '../../messages.json';

const comment = {
  id: 1,
  email: 'cursos@platzi.com',
  name: 'Platzi Team',
  body: 'Este es un comentario de prueba',
};

test('Comment should render the component', () => {
  const tree = renderer.create(
    <IntlProvider locale="es" messages={messages.es}>
      <Comment {...comment} />
    </IntlProvider>,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});