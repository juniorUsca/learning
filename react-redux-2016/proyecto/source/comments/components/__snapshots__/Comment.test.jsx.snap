// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment should render the component 1`] = `
<article
  id="comment-1"
>
  <div>
    <span
      dangerouslySetInnerHTML={
        Object {
          "__html": "Por: <a href=\\"mailto:cursos@platzi.com\\" target=\\"_blank\\">Platzi Team</a>",
        }
      }
    />
  </div>
  <p>
    Este es un comentario de prueba
  </p>
</article>
`;
